www/js/overlay.js

```diff
@@ -278,12 +266,28 @@ const toggleAnnouncements = h(
   h('div.setting-label', 'Announcement'),
 );
 
+const urlText = h(
+  'div.copy-text',
+  h('text.url', {
+    innerText: `${url}`,
```

Comment:
when overlay is initially off we get this:
![image](https://user-images.githubusercontent.com/2982784/73759469-dff32180-4720-11ea-913a-856a86aadd52.png)
When it gets turned on the URL doesn't update and it still has `undefined` as the port. 

Can you add an update to the url when overlay is turned on. Also when it is off, maybe hide it?